drivers/hdc1000/hdc1000.c

```diff
@@ -33,6 +33,14 @@
 
 #define I2C_SPEED                  I2C_SPEED_FAST
 
+#ifndef HDC1000_RENEW_INTERVAL
+#define HDC1000_RENEW_INTERVAL     1000000ul
```

Comment:
I think the default style is to put parenthesis around numerals for defines

Reply:
Got it :)